docs/product-spec.md

```diff
@@ -0,0 +1,104 @@
+# VxMail Product Spec
+
+## Background & Motivation
+
+The COVID-19 pandemic presents a significant challenge to the US
+General Election of 2020. How can we safely hold elections when it may
+be unsafe to congregate in a typical precinct? There is an emerging
+consensus that one important solution will be the expansion of
+vote-by-mail.
+
+Our product hypothesis is that, in states that currently have little
+vote-by-mail, the small-to-medium sized counties that process no more
+than 50,000 votes are likely to have the hardest time making the
+necessary adjustments. Many of those counties are lightly staffed, and
+many have a completely manual process for processing absentee ballot
+requests, stuffing and mailing ballots out, and finally receiving,
+sorting, verifying, extracting, and scanning the returned ballots. A
+manual process that works for a few hundred ballots quickly falls over
+for a few thousand ballots, and establishing a more automated process
+in the 6 months we have left is a huge challenge for each small county
+to undertake.
+
+We propose VxMail, a set of tools to help small counties quickly scale
+up their vote-by-mail capacity.
+
+## Key Hypothesis
+
+A number of small and some medium-sized counties have a sufficiently
+manual process for a relatively small number of vote-by-mail ballots
+today. COVID-19 will cause, for them, a shift to vote-by-mail, with 10
+times more ballots, which makes the manual process no longer
+practical&mdash;because of insufficient speed and accuracy. With
```

Comment:
Reduce number of commas.
```suggestion
Many small and medium-sized counties currently have a sufficient
manual process for a relatively small number of vote-by-mail ballots.
Due to COVID-19, these counties will see 10 times the volume of 
vote-by-mail ballots. This increase will overwhelm their current 
process due to insufficient speed and accuracy. With
```

Reply:
reduced commas with slightly different wording in upcoming push.